### ISSUE TITLE
processing_time should not be smallint

### ISSUE THREAD

**Issue by cleder** [ofa/django-bouncy#20]
`
 File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django_bouncy/views.py", line 128, in endpoint
    return process_message(message, data)
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django_bouncy/views.py", line 149, in process_message
    return process_delivery(message, notification)
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django_bouncy/views.py", line 258, in process_delivery
    smtp_response=delivery['smtpResponse']
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django/db/models/manager.py", line 127, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django/db/models/query.py", line 348, in create
    obj.save(force_insert=True, using=self.db)
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django/db/models/base.py", line 734, in save
    force_update=force_update, update_fields=update_fields)
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django/db/models/base.py", line 762, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django/db/models/base.py", line 846, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django/db/models/base.py", line 885, in _do_insert
    using=using, raw=raw)
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django/db/models/manager.py", line 127, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django/db/models/query.py", line 920, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 974, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django/db/utils.py", line 98, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/local/django/virtualenvs/mysite/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
DataError: smallint out of range
`

**Comment by cleder** [comment]
https://code.djangoproject.com/ticket/20446 => this cannot be tested with sqlite as smallin = integer on sqlite

**Comment by psnma** [comment]
This error happened to me 2 hours ago.

## DataError
### smallint out of range

**django_bouncy/views.py in process_delivery at line 258**
```
            mail_from=mail['source'],
            address=eachrecipient,
            # delivery
            delivered_time=delivered_datetime,
            processing_time=int(delivery['processingTimeMillis']),
            smtp_response=delivery['smtpResponse']  # 258
        )]
    # Send signals for each delivery.
    for eachdelivery in deliveries:
        signals.feedback.send(
```

'Message': '{ "notificationType":"Delivery", 'delivery': { 'processingTimeMillis': 44980, ...

**Comment by nickcatal** [comment]
Closed in #21
